lib/rules/no-debug.ts

```diff
@@ -120,7 +126,19 @@ export default createTestingLibraryRule<Options, MessageIds>({
           }
         );
 
-        if (isDebugUtil || isDeclaredDebugVariable || isChainedReferenceDebug) {
+        const isVariableFromBuiltInConsole = builtInConsoleNodes.some(
+          (variableDeclarator) => {
+            const variables = context.getDeclaredVariables(variableDeclarator);
+            return variables.some(
+              ({ name }) => name === callExpressionIdentifier.name
```

Comment:
I'm not sure if this is too brittle. Can you add an invalid test case mixing both console and Testing Library `debug`? to make sure they work fine together. Something like:

```javascript
import { debug as testingLibraryDebug } from '@testing-library/package'

test('invalid scenario: mixing console and Testing Library debug', () => {
  const { debug } = console
  debug()
  console.debug()
  testingLibraryDebug() // <-- this is the only one reported
})
```

Reply:
added here https://github.com/testing-library/eslint-plugin-testing-library/pull/349/commits/1778fa8ed6771e4869acbe0de254e1435cecab2a I had to add an extra check to make it work. Nice catch! I went too easy on the check 😆  👏